website/docs/profile.md

```diff
@@ -0,0 +1,28 @@
+Enable or disable instrumented performance profiling support for binaries.
+
+```lua
+profile "value"
+```
+
+### Parameters ###
+| Value   | Description                                                             |
+-------------------------------------------------------------------------------------
+| Default | Use the toolset's default instrumentated performance profiling setting. |
+| On      | Turn on instrumented performance profiling.                             |
+| Off     | Turn off instrumented performance profiling.                            |
+
+### Applies To ###
+
+Project configurations where the `kind` is an executable or a shared library.
```

Comment:
What happens to static libraries (used by those "executables")?
I assume that profiling instrumentation should be added (at least for gcc/clang), no?